.github/workflows/codeql.yml

```
@@ -19,7 +19,8 @@ jobs:
         language: [go]
     steps:
       - name: Checkout
-        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5@11bd71901bbe5b1630ceea73d27597364c9af683 # v4- name: Initialize CodeQL
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        name: Initialize CodeQL
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)